modules/oadp-creating-backup-cr.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+[id="oadp-creating-backup-cr_{context}"]
+= Creating a Backup CR
+
+You back up Kubernetes images, internal images, and persistent volumes (PVs) by creating a `Backup` custom resource (CR).
+
+.Prerequisites
+
+* You must install the OpenShift API for Data Protection (OADP) Operator.
+* The `DataProtectionApplication` CR must be in a `Ready` state.
+* Backup prerequisites:
+** You must have S3 object storage configured for Velero.
```

Comment:
Maybe this is my ignorance. Do we need to mention what Velero is and how to determine if everything is configured properly?